rusk/src/bin/main.rs

```diff
@@ -181,7 +195,53 @@ async fn startup_with_tcp_ip(
 
     let rusk = Rusk::default();
     let keys = KeysServer::new(rusk);
+    let network = NetworkServer::new(kadcast);
 
     // Build the Server with the `Echo` service attached to it.
-    Ok(Server::builder().add_service(keys).serve(addr).await?)
+    Ok(Server::builder()
+        .add_service(keys)
+        .add_service(network)
+        .serve(addr)
+        .await?)
+}
+
+/// Setup clap to handle kadcast network configuration
+fn network_config<'a>(app: App<'a, 'a>) -> App<'a, 'a> {
+    app.arg(
+        Arg::with_name("kadcast_public_address")
+            .long("kadcast_public_address")
+            .help("Public address you want to be identified with. Eg: 193.xxx.xxx.198:9999")
```

Comment:
Can we put in the description the difference between the public address and the listen address?